asllib/doc/ASLTypingReference.tex

```diff
@@ -4753,6 +4809,30 @@ \section{TypingRule.EGetBitFieldTyped \label{sec:TypingRule.EGetBitFieldTyped}}
 
 \isempty{\subsection{Comments}}
 
+\section{TypingRule.EGetTupleItem \label{sec:TypingRule.EGetTupleItem}}
+
+\subsection{Prose}
+All of the following apply:
```

Comment:
Do we need the itemize here? It looks like there’s only one clause?

Reply:
the list is not done. I need to go back and fill it.